variables.tf

```
@@ -227,6 +227,12 @@ variable "enable_stream_logs" {
   description = "Whether to create groups in CloudWatch Logs for proxy and deployment logs, and stream logs from each instance in your environment"
 }
 
+variable "enable_elb_logs" {
```

We have begun using this convention
```suggestion
variable "elb_logs_enabled" {
```